src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/ProbationRecord.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.justice.probation.courtcaseservice.service.model;
 
+import java.util.List;
```

Comment:
Not sure how this has crept in twice 🤔

Reply:
Weird - I have auto organise on for that. Done anyway.